census/README.md

```diff
@@ -1,313 +1,324 @@
-See options below:
+<h1>Training</h1>
 
-### Using local python
-Run the code on your local machine:
+* **GCloud configuration:**
 
 ```
 export TRAIN_STEPS=1000
 DATE=`date '+%Y%m%d_%H%M%S'`
-export OUTPUT_DIR=census_$DATE
-rm -rf $OUTPUT_DIR
+export JOB_DIR=census_$DATE
+rm -rf $JOB_DIR
+export TRAIN_FILE=census_data/adult.data.csv
+export EVAL_FILE=census_data/adult.test.csv
 ```
 
-### Model location reuse
-Its worth calling out that unless you want to reuse the old model output dir,
-model location should be a new location so that old model doesn't conflict with new
-one. 
+* **Test locally:**
 
 ```
 python -m trainer.task --train-files $TRAIN_FILE \
-                       --eval-files $EVAL_FILE \
-                       --job-dir $OUTPUT_DIR \
-                       --train-steps $TRAIN_STEPS \
-                       --eval-steps 100
+    --eval-files $EVAL_FILE \
+    --job-dir $JOB_DIR \
+    --train-steps $TRAIN_STEPS \
+    --eval-steps 100
 ```
 
-### Using gcloud local
-Run the code on your local machine using `gcloud`. This allows you to mock
-running it on the cloud:
+* **Google Cloud ML Engine**
+
+* **GCloud configuration:**
 
 ```
-export TRAIN_STEPS=1000
 DATE=`date '+%Y%m%d_%H%M%S'`
-export OUTPUT_DIR=census_$DATE
-rm -rf $OUTPUT_DIR
+export JOB_DIR=census_$DATE
+rm -rf $JOB_DIR
+export TRAIN_FILE=gs://cloud-samples-data/ml-engine/census/data/adult.data.csv
+export EVAL_FILE=gs://cloud-samples-data/ml-engine/census/data/adult.test.csv
+export TRAIN_STEPS=1000
 ```
 
+* **Run locally:**
+
 ```
 gcloud ml-engine local train --package-path trainer \
-                           --module-name trainer.task \
-                           -- \
-                           --train-files $TRAIN_FILE \
-                           --eval-files $EVAL_FILE \
-                           --job-dir $OUTPUT_DIR \
-                           --train-steps $TRAIN_STEPS \
-                           --eval-steps 100
+    --module-name trainer.task \
+    -- \
+    --train-files $TRAIN_FILE \
+    --eval-files $EVAL_FILE \
+    --job-dir $JOB_DIR \
+    --train-steps $TRAIN_STEPS \
+    --eval-steps 100
 ```
 
-### Using Cloud ML Engine
-*NOTE* If you used downloaded the training files to your local file system, be sure
+* **Run in Google Cloud ML Engine**
+
+You can train the model on Cloud ML Engine:
+
+*NOTE:* If you downloaded the training files to your local filesystem, be sure
 to reset the `TRAIN_FILE` and `EVAL_FILE` environment variables to refer to a GCS location.
 Data must be in GCS for cloud-based training.
 
 Run the code on Cloud ML Engine using `gcloud`. Note how `--job-dir` comes
 before `--` while training on the cloud and this is so that we can have
 different trial runs during Hyperparameter tuning.
 
+* **GCloud configuration:**
+
 ```
 DATE=`date '+%Y%m%d_%H%M%S'`
 export JOB_NAME=census_$DATE
-export GCS_JOB_DIR=gs://<my-bucket>/path/to/my/jobs/$JOB_NAME
+export GCS_JOB_DIR=gs://your-bucket-name/to/my/jobs/$JOB_NAME
```

Comment:
`path` might be worth keeping. Though I prefer we just suggest a real name to use.

Reply:
ack